games-action/openclonk-7.0

```diff
@@ -0,0 +1,140 @@
+SUMMARY="A free multiplayer action game where you control clonks"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you\
+control clonks, small, but witty and nimble humanoid beings. The game\
+is mainly about mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org/"
+COPYRIGHT="
+2009-2016 The OpenClonk Team and contributors
+2006-2007  Alexander Post
+2004-2013  Armin Burgmeier
+2005-2006, 2008-2010  Asmageddon
+2010, 2012  Benjamin Herr
+2006, 2012  Bernhard Bonigl
+2009-2010  Carl-Philip Hänsch
+2001  Carlo Teubner
+2009, 2011-2013  David Dormagen
+2011-2012  Felix Wagner
+2006  Florian Groß
+2004-2012  Günther Brammer
+2006-2008  Julian Raschke
+2010-2012  Julius Michaelis
+2010-2011  Maikel de Vries
+2010  Manuel Riecke
+2009  Mark Haßelbusch
+2009-2012  Martin Plicht
+1998-2009  Matthes Bender
+2001  Michael Käser
+2009  mizipzor
+2009-2013  Nicolas Hake
+2013  Oliver Schneider
+2010  Peewee
+2001-2011  Peter Wortmann
+2012  Philipp Kern
+2009-2010  Richard Gerum
+2001-2013  Sven Eberhardt
+2004-2005, 2009-2013  Tobias Zwick
+"
```

Comment:
Can you please add one tab in front of each line, from line 7 to line 35?
With vi you can add one tab on several lines with only a few key presses. Here you would tipe: `7G29>>`
After that you just need to join lines 35+36 and 7+8. Here's a diff:
```diff
 HOMEPAGE="http://www.openclonk.org/"
-COPYRIGHT="
-2009-2016 The OpenClonk Team and contributors
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors
-2006-2007  Alexander Post
+	2006-2007 Alexander Post
-2004-2013  Armin Burgmeier
+	2004-2013  Armin Burgmeier
-2005-2006, 2008-2010  Asmageddon
+	2005-2006, 2008-2010  Asmageddon
-2010, 2012  Benjamin Herr
+	2010, 2012  Benjamin Herr
-2006, 2012  Bernhard Bonigl
+	2006, 2012  Bernhard Bonigl
-2009-2010  Carl-Philip Hänsch
+	2009-2010  Carl-Philip Hänsch
-2001  Carlo Teubner
+	2001  Carlo Teubner
-2009, 2011-2013 David Dormagen
+	2009, 2011-2013  David Dormagen
-2011-2012  Felix Wagner
+	2011-2012  Felix Wagner
-2006  Florian Groß
+	2006  Florian Groß
-2004-2012  Günther Brammer
+	2004-2012  Günther Brammer
-2006-2008  Julian Raschke
+	2006-2008  Julian Raschke
-2010-2012  Julius Michaelis
+	2010-2012  Julius Michaelis
-2010-2011  Maikel de Vries
+	2010-2011  Maikel de Vries
-2010  Manuel Riecke
+	2010  Manuel Riecke
-2009  Mark Haßelbusch
+	2009  Mark Haßelbusch
-2009-2012  Martin Plicht
+	2009-2012  Martin Plicht
-1998-2009  Matthes Bender
+	1998-2009  Matthes Bender
-2001  Michael Käser
+	2001  Michael Käser
-2009  mizipzor
+	2009  mizipzor
-2009-2013  Nicolas Hake
+	2009-2013  Nicolas Hake
-2013  Oliver Schneider
+	2013  Oliver Schneider
-2010  Peewee
+	2010  Peewee
-2001-2011  Peter Wortmann
+	2001-2011  Peter Wortmann
-2012  Philipp Kern
+	2012  Philipp Kern
-2009-2010  Richard Gerum
+	2009-2010  Richard Gerum
-2001-2013  Sven Eberhardt
+	2001-2013  Sven Eberhardt
-2004-2005, 2009-2013  Tobias Zwick
-"
+	2004-2005, 2009-2013  Tobias Zwick"
 LICENSE="ISC"
```